components/tools/OmeroWeb/omeroweb/webclient/templates/webclient/annotations/metadata_acquisition.html

```diff
@@ -70,6 +61,49 @@
 
                 $('.can-collapse.defclose').removeClass('defclose').addClass('closed').next().hide();
 
+                // Load Original Metadata...
+                var origMetadataLoaded = false;
+                function loadOriginalMetadata () {
+                    if (!origMetadataLoaded) {
+                        origMetadataLoaded = true;
+                        var loadingHtml = '<img alt="Loading" src="{% static "webgateway/img/spinner.gif" %}">'
+                                        + 'Loading Original Metadata...'
+                        var url = $("#orig_metadata_link").prop('href');
+                        $("#orig_metadata")
+                            .html(loadingHtml)
+                            .load(url, function( response, status, xhr ) {
+                                if (xhr.status == 408) {
+                                    origMetadataLoaded = false;
+                                    $("#orig_metadata").html("Timeout. <a href='#' class='load_original_metadata'>Try again</a>");
+                                } else {
+                                    // setup quick search fields (but not if MANY rows, since it crashes!)
+                                    if ($("#global_metadata tr").length < 100000) {
```

Comment:
@joshmoore This is really just to catch the 'edge case' that @gusferguson was testing with. I haven't tried to test the performance of the quicksearch plugin with various different list sizes, but certainly could do.